docs/arlas-iam.md

```diff
@@ -0,0 +1,224 @@
+| ARLAS_AUTH_KEYCLOAK_REALM    | arlas_auth.keycloak.realm              | arlas                                                     | Keycloak        |
+| ARLAS_AUTH_KEYCLOAK_URL      | arlas_auth.keycloak.auth-server-url    | http://keycloak:8080/auth                                 | Keycloak        |
+| ARLAS_AUTH_KEYCLOAK_RESOURCE | arlas_auth.keycloak.resource           | arlas                                                     | Keycloak        |
+| ARLAS_AUTH_KEYCLOAK_SECRET   | arlas_auth.keycloak.credentials.secret | none                                                      | Keycloak        |
+
+## Protection rules
+ARLAS IAM allows the enforcement of two kinds of protection mechanisms. They are achieved by the definition of rules and
+(HTTP) headers that are collected by the *Policy Enforcer* and transferred to the ARLAS backend component it protects.  
+They are strings of characters with a specific formatting that are expected to be found in the access token and/or permission token
+(RPT: requesting party token) in specific claims.
+
+### Protection of ARLAS WUI dashboards: groups
+By default dashboards are only viewable and editable by their creator (owner).  
+One can share (view and/or edit rights) a dashboard with any group of users they already belong to,
+e.g. if `userA` belongs to groups `grp1` and `grp2`, they can share their dashboards with part or all of these groups,
+and these groups only.  
+It implies that prior to sharing a dashboard, groups must be created and assigned to users.  
+This is done by assigning specific roles whose names are formatted as `group/config.json/GRPNAME`,
+e.g. `group/config.json/spot6`(in this example, the group name is `spot6` and will be displayed as such in ARLAS hub).
+A good practice is to assign data protection headers to these roles in order to enforce an even better protection level, e.g.:
+- `group/config.json/spot6`
+    - `h:column-filter:spot6_*:*`
+
+### Protection of ARLAS APIs: rules
+The actions a user can do, i.e. API endpoints and HTTP verbs, can be limited to a configurable list of URIs.
+The expected format is `rule:resource:verbs` or `r:resource:verbs`. The `resource` part of the rule is used as a regex
+to match a requested URI.
+Examples:
+- `r:explore/.*:GET,POST`
+- `r:collections/.*:GET`
+
+These rules do not need to be configured as they are already associated to default roles defined in the IAM module.  
+These roles are:
+- `role/arlas/user` (rules to view data)
+- `role/arlas/tagger` (rules to use the Tagger backend)
+- `role/arlas/builder` (rules to create/edit/delete ARLAS WUI dashboards)
+- `role/arlas/owner` (rules to manage collections in ARLAS server and organisations/users in ARLAS IAM server)
+- `role/arlas/importer` (rule to import collections via the dedicated ARLAS server endpoint, mainly used by M2M processes)
```

Comment:
rules to be consistent with the other lines ?

Reply:
indeed. Done.